Some docs gardening (#3544)
akka/akka-http: docs/src/main/paradox/implications-of-streaming-http-entity.md

### Consuming the HTTP Response Entity (Client)

> There's two use-cases to consume the entity of a response:

Reviewer: ```suggestion
There are two use-cases to consume the entity of a response:
```

Java
:   @@snip [HttpServerExampleDocTest.java]($test$/java/docs/http/javadsl/server/HttpServerExampleDocTest.java) { #consume-entity-directive }

> You can also access the raw dataBytes and run the underlying stream. For example, you could pipe the raw dataBytes into a

Reviewer: ```suggestion
You can also access the raw `dataBytes` and run the underlying stream. For example, you could pipe the raw `dataBytes` into a
```

### Discarding the HTTP Request Entity (Server)

You may want to discard the uploaded entity. For example, depending on some validation (e.g. "is user authorized to upload files?").

Please note that "discarding the HTTP Request Entity" means that the entire upload will proceed, even though you are not interested in the data
being streamed to the server. This is useful if you are simply not interested in the entity. 

> In order to discard the dataBytes explicitly you can invoke the `discardEntityBytes` bytes of the incoming `HttpRequest`:

Reviewer: ```suggestion
In order to discard the `dataBytes` explicitly you can invoke the `discardEntityBytes` bytes of the incoming `HttpRequest`:
```
